### ISSUE TITLE
The result is far away from paper while using ae test

### ISSUE THREAD

**Issue by zywbupt** [wuxiaofei01/FastGrasp#2]
hello. Thank you for your code. But when I  When I reproduced the results of the paper through your code, I found that the results of using only the ae module to test were different from the results of the smile experiment in the paper. The following table is the results of the ae module that I recorded through your code. I don’t know if it is my config file. The following is my config file. I don’t know if there is a problem with my config configuration that caused the difference in results. I look forward to your reply. Thank you.

![Image](https://github.com/user-attachments/assets/f6bfdcf4-79bf-4d8e-b042-564de7325c9c)
it is grab/eval_ae.json
{
    "seed": 0,
    "batch_size": 1,
    "use_cuda": 1,
    "dataloader_workers": 1,
    "train_mode": "Test",
    "dataset": "grab",
    "mano":"oakink",
    "encoder_layer_sizes": [
        1024,
        2048,
        1024,
        768
    ],
    "decoder_layer_sizes": [
        2048,
        1024,
        256,
        61
    ],
    "intention":false,
    "obj_inchannel": 4,
    "loss_type": "CD",
    "model_path" : "checkpoints/ae.pth"
}
it is oakink/eval_ae.json
{
    "seed": 0,
    "batch_size": 1,
    "use_cuda": 1,
    "dataloader_workers": 1,
    "train_mode": "Test",
    "dataset": "oakink",
    "mano": "oakink",
    "encoder_layer_sizes": [
        1024,
        2048,
        1024,
        768
    ],
    "decoder_layer_sizes": [
        2048,
        1024,
        256,
        61
    ],
    "intention" :false,
    "obj_inchannel": 4,
    "loss_type": "CD",
    "model_path" : "checkpoints/ae.pth"
}

**Comment by wuxiaofei01** [comment]
Can you visualize the mesh? I suspect it is possible that you are loading the wrong mano file。

**Comment by zywbupt** [comment]
![Image](https://github.com/user-attachments/assets/3523122e-22f1-4d44-9143-dec6c2ff2197)

![Image](https://github.com/user-attachments/assets/85801f37-5ced-4402-86f5-1e6ff2dc08ca)

![Image](https://github.com/user-attachments/assets/a73da8c9-a247-4ead-8c26-12ecca4c5cba)
these are some of  saved ply results

**Comment by wuxiaofei01** [comment]
The visualization result is indeed an output that conforms to the whole, but there are some problems with the details. Please could you check if oakink's mano layer is used or also switch to grab in config to try it.

**Comment by wuxiaofei01** [comment]
> The visualization result is indeed an output that conforms to the whole, but there are some problems with the details. Please could you check if oakink's mano layer is used or also switch to grab in config to try it.

Please make sure the mano layer is formatted correctly. If you have any questions, please leave a message.

**Comment by wuxiaofei01** [comment]
Ok, it is recommended to follow readme.md, and you can leave a message if you have any questions.

**Comment by wuxiaofei01** [comment]
Could you please test the results of the oakink dataset? Because my checkpoit is supposed to be trained on oakink dataset.

**Comment by zywbupt** [comment]
> Could you please test the results of the oakink dataset? Because my checkpoit is supposed to be trained on oakink dataset.
Thank you for you quick reply. For oakink is too large to test. so I want to do some further work based on grab at first. i wonder if you can provide the data preprocess code for grab and the trained grab checkpoint for email 772725101@qq.com. Thank you very much !!!
